### DISCUSSION TITLE
Android 的 pag 缓存目录路径是什么

### DISCUSSION THREAD

**Discussion by DXBbeginner** [Tencent/libpag#2395] [Q&A] [answered]
如题 求助

**Comment by domchen** [comment, marked as answer]
就是App沙箱的cache目录。具体代码在 PAGDiskCache.java 里：https://github.com/Tencent/libpag/blob/main/android/libpag/src/main/java/org/libpag/PAGDiskCache.java#L56

会在这个目录下写入一个 cache.cfg文件，以及一个files/目录。
